README.md

```diff
@@ -32,16 +32,16 @@ You are now ready to start the server, run
 There are 3 major segments of the applications API
 
 1. The Authentication / Authorization controller - to handle functions such as login / register / logout / token rotation
-2. The user controller - to handle retrieval of user information
-3. The wallet controller - to handle wallet topup and transfer of funds to other users
+2. The user controller - to handle the retrieval of user information
+3. The wallet controller - to handle wallet top-up and transfer of funds to other users
 
 
--  ### **Lets get started with Auth**
+-  ### **Let's get started with Auth**
 
 	**Registration**
 	> POST - https://p2p-api.onrender.com/api/auth/register
 
-	This route expects a payload of the fomat 
+	This route expects a payload of the format 
```

Comment:
**Suggestion:** Correct the grammar in line 44 from 'a payload of the format' to 'a payload in the following format'.
```suggestion
 This route expects a payload in the following format
```